README.md

```diff
@@ -47,8 +47,8 @@ mix test
 
 # Copyright and License
 
-Copyright 2019, [Software Mansion](https://swmansion.com/?utm_source=git&utm_medium=readme&utm_campaign=elixir-fdk-aac-docker)
+Copyright 2019, [Software Mansion](https://swmansion.com/?utm_source=git&utm_medium=readme&utm_campaign=docker-ubuntu18-membrane)
 
-[![Software Mansion](https://membraneframework.github.io/static/logo/swm_logo_readme.png)](https://swmansion.com/?utm_source=git&utm_medium=readme&utm_campaign=elixir-fdk-aac-docker)
+[![Software Mansion](https://membraneframework.github.io/static/logo/swm_logo_readme.png)](https://swmansion.com/?utm_source=git&utm_medium=readme&utm_campaign=docker-ubuntu18-membrane
```

Comment:
ubuntu18 is ambiguous. I would suggest using its codename (`bionic`)

Reply:
I may use `ubuntu1804`

Reply:
That seems off - ubuntu versions, especially in such use cases are referred by their codename, like in build environments in Travis https://docs.travis-ci.com/user/reference/xenial/

Reply:
But I won't remember what is bionic in a few months

Reply:
And most DockerHub repositories follow the numeric convention

Reply:
Most? How have you measured that? From what I've seen they either support both (https://hub.docker.com/_/buildpack-deps) or only codenames (especially for Debian https://hub.docker.com/_/node)

Reply:
And if you insist on using numbers, make it "ubuntu-18.04-membrane"

Reply:
Yes I have :D
https://hub.docker.com/search?q=ubuntu18&type=image
https://hub.docker.com/search?q=bionic&type=image
18.04 can be

Reply:
Have you noticed the popularity of images under both links? :stuck_out_tongue:

Reply:
Have you noticed that the most popular is and image of PHP? :D

Reply:
ok, I can change to this bionic... but that bionic-membrane or ubuntu-bionic-membrane?